reference/test/matrix/dense_kernels.cpp

```diff
@@ -2009,6 +2009,74 @@ TYPED_TEST(Dense, ExtractsDiagonalFromNonSquareMatrix)
 }
 
 
+TYPED_TEST(Dense, InplaceAbsolute)
+{
+    using T = typename TestFixture::value_type;
+    // clang-format off
+    // {1.0, -1.0, -0.5},
+    // {-2.0, 2.0, 4.5},
+    // {2.1, 3.4, 1.2}
+    // clang-format on
+    this->mtx5->compute_absolute_inplace();
+
+    GKO_ASSERT_MTX_NEAR(this->mtx5,
+                        l({{1.0, 1.0, 0.5}, {2.0, 2.0, 4.5}, {2.1, 3.4, 1.2}}),
+                        r<TypeParam>::value);
+}
+
+
+TYPED_TEST(Dense, InplaceAbsoluteSubMatrix)
+{
+    using T = typename TestFixture::value_type;
+    // mtx takes the left top corner 2 x 2 matrix
+    auto mtx = this->mtx5->create_submatrix(gko::span{0, 2}, gko::span{0, 2});
+    // clang-format off
+    // {1.0, -1.0, -0.5},
+    // {-2.0, 2.0, 4.5},
+    // {2.1, 3.4, 1.2}
+    // clang-format on
```

Comment:
nit: Is this really needed in every test?

Reply:
I am not pretty sure. I keep it because this file usually show the matrix in each test